reference/import_export/formats/product.rst

```diff
@@ -0,0 +1,33 @@
+Product CSV data structure
+==========================
+
+Products are exported in a CSV file with the following fields:
+
+Attributes fields can be one of these:
```

Comment:
You should add all fields: sku is required, but you can also add categories (comma separated) and enabled